docs/master/eloquent/complex-where-conditions.md

```diff
@@ -322,3 +322,52 @@ This query would retrieve all persons, no matter if they have a role or not:
 You may register a custom `\Nuwave\Lighthouse\WhereConditions\Operator` through a service provider.
 This may be necessary if your database uses different SQL operators then Lighthouse's default or you
 want to extend/restrict the allowed operators.
+
+First create a class that implements `\Nuwave\Lighthouse\WhereConditions\Operator`, e.g. `\App\GraphQL\CustomSQLOperator`.
+In following example we will extend `\Nuwave\Lighthouse\WhereConditions\SQLOperator` that actually implements 
+`\Nuwave\Lighthouse\WhereConditions\Operator` and modify result of `enumDefinition()` to include `ILIKE` operator to the
+end of the `SQLOperator` enum. 
+
+```php
+use Nuwave\Lighthouse\WhereConditions\SQLOperator;
+
+class CustomSQLOperator extends SQLOperator
```

Comment:
I think this is quite fragile, so i would not recommend doing this to users.

Reply:
Ye... ok. do you have an example for me? :D

Reply:
We have been discussing some ways that allow extending the `SQLOperator` more cleanly, let's hold off on our recommended solution until we implemented something better.

Doing string manipulation on the definition string is really hacky.